checker/bin/wpi-annotation-paths.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Given the name of a directory containing the results produced by an
+# invocation of wpi-many.sh and a filename, output the paths to the .ajava
+# files produced by whole-program inference to a .txt file with the given name.
+
+# Usage:
+#   wpi-annotation-paths TARGETDIR RESULT_FILE_NAME
+
+TARGETDIR="$1"
+RESULT_FILE_NAME="$2"
+
+if [ "$#" -ne 2 ]; then
+  echo "Usage: $(basename "$0") TARGETDIR RESULT_FILE_NAME" >&2
+  exit 1
+fi
+
+for wpi_log_file in "$TARGETDIR"/*-wpi-stdout.log;
```

Comment:
It would be good to check that there are any files that match this pattern before starting this loop, and fail with an error message indicating that the wrong directory was probably provided if not.